src/middleware/nunjucksLoader.middleware.ts

```diff
@@ -57,5 +58,7 @@ export default class NunjucksLoader {
     app.locals.pageTitleSuffix = PAGE_TITLE_SUFFIX
 
     app.locals.nonce = nonce
+
+    app.locals.payByAccountEnabled = this.PAY_BY_ACCOUNT_FEATURE_ENABLED
```

Comment:
Being picky, but can we make the nunjucks variable `payByAccountFeatureEnabled` as well?

Reply:
Done